lldb/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ if(POLICY CMP0075)
   cmake_policy(SET CMP0075 NEW)
 endif()
 
+if(POLICY CMP0077)
```

Comment:
I think it would be nice to add (upstream) comments here that explain what these policies are and why we enable them.

Reply:
I should have done that. Swift's cmake has comments, but lldb's didn't and I blindly followed the existing policy config. I'll do a follow up.